### ISSUE TITLE
change OpenNetconf to allow custom Exec and Open cmd

### ISSUE THREAD

**Issue by hellt** [scrapli/scrapligo#45]
to be consistent with a regular Open we may want to allow users to override exec and open cmds

**Comment by carlmontanari** [comment]
good call, shoulda done that in the first place 😁 will get that sorted later this week!
